entrypoint.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-htmlproofer "${INPUT_DIRECTORY}"
\ No newline at end of file
+htmlproofer ${INPUT_ARGUMENTS} "${INPUT_DIRECTORY}"
```

Comment:
If you're worried about it failing, then you can use `${INPUT_ARGUMENTS:-""}`.

Reply:
An unset variable shouldn’t be a problem with the currently used `set -e` but would throw an error if that is ever changed to `set -eu`

I figured that specifying the default in `action.yml` was a good way to avoid that, but using `${INPUT_ARGUMENTS:-""}` works too!

Reply:
>An unset variable shouldn’t be a problem with the currently used `set -e` but would throw an error if that is ever changed to `set -eu`

Good to know, thanks!
If that's the case then there's no need to set a default.

Also, the reason why I don't want it set in the `action.yml`, is because it's confusing...